Tests/LibSQL/TestSqlStatementExecution.cpp

```diff
@@ -114,6 +114,19 @@ TEST_CASE(insert_wrong_number_of_values)
     EXPECT(result->inserted() == 0);
```

Comment:
(Commenting on the commit message here)
Commits should be written in an imperative mood, see: https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy
So instead of `LibSQL: Added an INSERT without column names test`, should be more like `LibSQL: Add a test for INSERT statements without column names`.

Reply:
Thanks for the notice... fixed that!